@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcSAS.sqf

```diff
@@ -0,0 +1,63 @@
+params ["_heli", "_deltaTime"];
+#include "\fza_ah64_sfmplus\headers\core.hpp"
+
+private _pidSASPitch = _heli getVariable "fza_sfmplus_pid_sas_pitch";
+//_pidSASPitch set ["kp", P_KP];
+//_pidSASPitch set ["ki", P_KI];
+//_pidSASPitch set ["kd", P_KD];
+private _pidSASRoll  = _heli getVariable "fza_sfmplus_pid_sas_roll";
+//_pidSASRoll set ["kp", R_KP];
+//_pidSASRoll set ["ki", R_KI];
+//_pidSASRoll set ["kd", R_KD];
+private _pidSASYaw   = _heli getVariable "fza_sfmplus_pid_sas_yaw";
+//_pidSASRoll set ["kp", Y_KP];
+//_pidSASRoll set ["ki", Y_KI];
+//_pidSASRoll set ["kd", Y_KD];
+
+
+([_heli, fza_ah64_sfmplusEnableWind] call fza_sfmplus_fnc_getVelocities)   //<-- needs wind enabled flag
+    params [ 
+             "_gndSpeed"
+           , "_vel2D"
+           , "_vel3D"
+           , "_vertVel"
+           , "_velModelSpace"
+           , "_angVelModelSpace"
+           , "_velWorldSpace"
+           , "_angVelWorldSpace"
+           ];
+
+(_angVelModelSpace)
+    params [
+             "_angVelX"
+           , "_angVelY"
+           , "_angVelZ"
+           ];
+
+private _SASPitchOutput = 0.0;
```

Comment:
`_sas`

Reply:
There are instances where SAS isn't the leading part of the text, keeping it SAS makes sense

Reply:
The general naming convention for abbreviations in code are to treat them as words, which means that generally they should be `Sas`, similar to what you have as `Np` below. Sometimes we do `SAS` in the middle but variables starting with capitals is inconsistent with our style

Reply:
the lack of consistency of sas annoys me.